pkgs/top-level/all-packages.nix

```diff
@@ -823,6 +823,13 @@ with pkgs;
 
   oauth2c = callPackage ../tools/security/oauth2c { };
 
+  octodns = python3Packages.callPackage ../tools/networking/octodns { };
+
+  octodns-providers = recurseIntoAttrs {
+    hetzner = python3Packages.callPackage ../tools/networking/octodns-providers/hetzner { };
+    powerdns = python3Packages.callPackage ../development/python-modules/octodns-powerdns { };
```

Comment:
The derivation path is off.

Reply:
fixed, I also changed the path octodns/providers